spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/controller/TaskDeploymentController.java

```diff
@@ -161,7 +112,7 @@ public void destroyTask(@PathVariable("name") String name) {
 	 * @param properties the runtime properties for the task, as a comma-delimited list of
 	 * 					 key=value pairs
 	 */
-	@RequestMapping(value = "/deployments/{name}", method = RequestMethod.POST)
+	@RequestMapping(value = "/{name}", method = RequestMethod.POST)
 	@ResponseStatus(HttpStatus.CREATED)
 	public void deploy(@PathVariable("name") String name, @RequestParam(required = false) String properties) {
 		TaskDefinition taskDefinition = this.repository.findOne(name);
```

Comment:
should wait to see what we decide wrt https://github.com/spring-cloud/spring-cloud-dataflow/pull/330 but something needs to be done here